src/dataprotection/azext_dataprotection/manual/aaz_operations/recovery_point.py

```diff
@@ -31,6 +36,10 @@ def _build_arguments_schema(cls, *args, **kwargs):
     def pre_operations(self):
         start_time = validate_recovery_point_datetime_format(self.ctx.args.start_time)
         end_time = validate_recovery_point_datetime_format(self.ctx.args.end_time)
+        if start_time and end_time:
+            import dateutil.parser
+            if dateutil.parser.parse(start_time) > dateutil.parser.parse(end_time):
+                logger.warning("The argument --start-time is greater than --end-time.")
```

Comment:
Do you need to raise error or not here?

Reply:
No, we only need to warn them. Such input is valid on service side and the query will return an empty object.